raiden/utils/__init__.py

```diff
@@ -175,7 +175,7 @@ def channel_to_api_dict(channel):
         "token_address": channel.token_address,
         "partner_address": channel.partner_address,
         "settle_timeout": channel.settle_timeout,
-        "balance": channel.contract_balance,
+        "balance": channel.distributable,
```

Comment:
If I do, I should also rename the API docs. 

But I actually think balance is a good name for this. The distributable balance of the token in the channel, is what a user would mostly think of as "Hmmm what is my balance in this channel?".